pkg/Microsoft.NETCore.Platforms/readme.md

```diff
@@ -0,0 +1,104 @@
+
+###RID-qualified assets are preferred
+NuGet will always prefer a RID-qualified asset over a RID-less asset.  For example if a package contains
+```
+lib/netcoreapp1.0/foo.dll
+runtimes/win/lib/netcoreapp1.0/foo.dll
+```
+NuGet will choose `runtimes/win/lib/netcoreapp1.0/foo.dll`.
+
+Additionally, NuGet will always prefer a RID-qualified asset over a RID-less asset, even if the framework is less specific for the RID-qualified asset.
+```
+lib/netstandard1.5/foo.dll
+runtimes/win/lib/netstandard1.0/foo.dll
+```
+When resolving for netstandard1.5/win7-x64 NuGet will choose `runtimes/win/lib/netstandard1.0/foo.dll` over `lib/netstandard1.5/foo.dll` even though `netstandard1.5` is more specific than `netstandard1.0`.
+
+### RID-qualified assets are never used for compile
+NuGet will select different compile-assets than runtime-assets.  The compile assets can never be RID-qualified.  Consider the package:
+```
+lib/netstandard1.5/foo.dll
+runtimes/win/lib/netstandard1.0/foo.dll
+```
+When resolving for netstandard1.5/win7-x64 will select `lib/netstandard1.5/foo.dll` for the compile asset and `runtimes/win/lib/netstandard1.0/foo.dll` for the runtime asset.
+
+##Adding new RIDs
+
+###Why do I need to add a new RID?
+NuGet's extensibility mechanism for platform-specific assets requires a RID be defined for any platform that needs assets specific to that platform.  Unlike TFMs, which have a known relationship in NuGet (eg net4.5 is compatible with net4.0), RIDs are opaque strings whigh NuGet knows nothing about.  The definition and relationship of RIDs comes solely from the `runtime.json` files within the root of the packages referenced by the project.
+As such, whenever we want to put a new RID in a project.json in order to get assets specific for that RID we have to define the rid in some package.  Typically that package is `Microsoft.NETCore.Platforms` if the RID is "official".  If you'd like to prototype you can put the RID in any other package and so long as that package is referenced you can use that RID.
+
+###Do I really need to add a new RID?
+If you're prototyping on a platform that is compatible with an existing platform then you can reuse the RID for that exsisting platform.  New RIDs are only needed when an asset needs to be different on a particular platform.
+
+`Microsoft.NETCore.Platforms` attempts to define all RIDs that packages may need, and as such will define RIDs for platforms that we don't actually cross compile for.  This is to support higher-level packages, 3rd party packages, that may need to cross-compile for that RID.
+
+###Naming convention
+We use the following convention in all newly-defined RIDs.  Some RIDs (win7-x64, win8-x64) predate this convention and don't follow it, but all new RIDs should follow it.
+`[os name].[version]-[architecture]-[additional qualifiers]`, for example `osx.10.10-x64` or `ubuntu.14.04-x64`
```

Comment:
For folks that want to augment the rid graph (either here or in their own package) I think we need to provide guidance on how they compute the RID for their platform.